bigquery_etl/alchemer/survey.py

```diff
@@ -149,9 +203,14 @@ def insert_to_bq(
 @click.option("--api_token", required=True)
 @click.option("--api_secret", required=True)
 @click.option("--destination_table", required=True)
-def main(date, survey_id, api_token, api_secret, destination_table):
+@click.option("--include_url_variables", required=True, default=False)
```

Comment:
I don't think this new option should be required, and I believe you need to indicate it's a boolean (either with `is_flag=True` or `type=bool`).
```suggestion
@click.option("--include_url_variables", is_flag=True, default=False)
```
If you do keep it required then you'll also need to update all the other ETLs that use this code.